mne/time_frequency/tfr.py

```diff
@@ -2428,6 +2428,10 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
         copy = baseline is not None
     data = rescale(data, times, baseline, mode, copy=copy)
 
+    if np.iscomplexobj(data):
+        logger.info('Converting phase-amplitude to power for visualization')
+        data = (data * data.conj()).real
```

Comment:
You are converting power here but for consistency you should leave it as amplitude for consistency (probably)
```suggestion
        data = np.sqrt((data * data.conj()).real)
```